tests/trsa

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash -e
+# Copyright (C) 2022 Simo Sorce <simo@redhat.com>
```

Comment:
2025

Reply:
I guess I should update all files I touch even if just moving things around? :)